client/src/views/workspaces/WorkspaceSharingModal.vue

```diff
@@ -6,20 +6,57 @@
       :title="I18n.valueAsTranslatable(workspaceName)"
       :close-button="{ visible: true }"
     >
-      <ms-report-text
-        :theme="MsReportTheme.Info"
-        v-if="isOnlyOwner()"
-        class="only-owner-warning"
+      <div
+        v-if="isSmallDisplay"
```

Comment:
There are missing conditions to this div: `isOnlyOwner() || (showCheckboxes && orgHasExternalUsers && isOpenReportTextContainer)` so it's not displayed when the info texts conditions aren't met. You might need to adapt the style when it's hidden as well (margin above the search bar seems too big)
![Capture d’écran 2025-04-30 à 15 44 21](https://github.com/user-attachments/assets/6eb6b429-2a9c-4069-aeeb-89d1f771eede)

Also, I'd suggest removing the small display condition, it would be a nice addition for the large display.